[2025-05-25T16:27:11.057+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: final_marketing_data_pipeline.generate_facts.contacts manual__2025-05-25T16:25:47.685943+00:00 [queued]>
[2025-05-25T16:27:11.092+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: final_marketing_data_pipeline.generate_facts.contacts manual__2025-05-25T16:25:47.685943+00:00 [queued]>
[2025-05-25T16:27:11.093+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-25T16:27:11.148+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): generate_facts.contacts> on 2025-05-25 16:25:47.685943+00:00
[2025-05-25T16:27:11.168+0000] {standard_task_runner.py:60} INFO - Started process 15949 to run task
[2025-05-25T16:27:11.178+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'final_marketing_data_pipeline', 'generate_facts.contacts', 'manual__2025-05-25T16:25:47.685943+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpzy5m0exo']
[2025-05-25T16:27:11.183+0000] {standard_task_runner.py:88} INFO - Job 112: Subtask generate_facts.contacts
[2025-05-25T16:27:11.433+0000] {task_command.py:423} INFO - Running <TaskInstance: final_marketing_data_pipeline.generate_facts.contacts manual__2025-05-25T16:25:47.685943+00:00 [running]> on host fe289fe396bd
[2025-05-25T16:27:11.754+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='yashbhawsar872@gmail.com' AIRFLOW_CTX_DAG_OWNER='Yash' AIRFLOW_CTX_DAG_ID='final_marketing_data_pipeline' AIRFLOW_CTX_TASK_ID='generate_facts.contacts' AIRFLOW_CTX_EXECUTION_DATE='2025-05-25T16:25:47.685943+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-25T16:25:47.685943+00:00'
[2025-05-25T16:27:11.757+0000] {test.py:39} INFO - ‚ñ∂Ô∏è Starting task: Generate Contacts
[2025-05-25T16:27:13.173+0000] {data_generation_new.py:125} INFO - ‚úÖ contacts_2025-05-25.csv generated (1000 rows)
[2025-05-25T16:27:13.464+0000] {metadata.py:109} INFO - üìù Logged stage 'generated' for contacts_2025-05-25.csv at 2025-05-25T16:27:13.177590
[2025-05-25T16:27:13.468+0000] {test.py:41} INFO - ‚úÖ Completed task: Generate Contacts
[2025-05-25T16:27:13.471+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-25T16:27:13.502+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=final_marketing_data_pipeline, task_id=generate_facts.contacts, execution_date=20250525T162547, start_date=20250525T162711, end_date=20250525T162713
[2025-05-25T16:27:13.697+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-25T16:27:14.394+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
